pkg/tests/make_rpm_test.py

```diff
@@ -174,6 +176,13 @@ def testBuild(self):
         # Make sure files exist.
         self.assertTrue(FileExists('test.rpm'))
 
+        found_arg = False
```

Comment:
Can you split the test so that there is one for --target and one without --target.